internal/manifests/manifestutils/tracing.go

```diff
@@ -0,0 +1,41 @@
+package manifestutils
+
+import (
+	"github.com/imdario/mergo"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
+)
+
+func PatchTracingJaegerEnv(tempo v1alpha1.TempoStack, dep *v1.Deployment) (*v1.Deployment, error) {
+	container := corev1.Container{
+		Env: []corev1.EnvVar{
+			{
+				Name:  "JAEGER_AGENT_HOST",
+				Value: "localhost",
+			},
+			{
+				Name:  "JAEGER_AGENT_PORT",
+				Value: "6831",
+			},
+			{
+				Name:  "JAEGER_SAMPLER_TYPE",
+				Value: "const",
+			},
+			{
+				Name:  "JAEGER_SAMPLER_PARAM",
+				Value: "1",
```

Comment:
Should this be configurable somehow? I'm not saying to do it for this PR, it is more because I'm curious about

Reply:
It depends how we want to use it. ^^

https://github.com/os-observability/tempo-operator/pull/299#discussion_r1135425230